x-pack/plugins/enterprise_search/public/applications/app_search/components/role_mappings/constants.ts

```diff
@@ -11,3 +11,32 @@ export const ROLE_MAPPINGS_TITLE = i18n.translate(
   'xpack.enterpriseSearch.appSearch.roleMappings.title',
   { defaultMessage: 'Role Mappings' }
 );
+
+export const DELETE_ROLE_MAPPING_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.deleteRoleMappingMessage',
+  {
+    defaultMessage:
+      'Are you sure you want to permanently delete this mapping? This action is not reversible and some users might lose access.',
+  }
+);
+
+export const ROLE_MAPPING_DELETED_MESSAGE = i18n.translate(
+  'xpack.enterpriseSearch.appSearch.roleMappingDeletedMessage',
+  {
+    defaultMessage: 'Successfully deleted Role mapping',
```

Comment:
Minor copy nit, Chris let us know earlier that App Search has a lot of instances where Title Casing should be using Sentence casing, so we've been keeping an eye out for it as we go along:

```suggestion
    defaultMessage: 'Successfully deleted role mapping',
```